environment-dev.yml

```
@@ -1,12 +1,11 @@
 name: modin
 channels:
   - conda-forge
-  - conda-forge/label/pandas_rc
 dependencies:
   - pip
 
   # required dependencies
-  - pandas==2.1.0rc0
+  - pandas>=2.1,<2.2
```

```suggestion
  - pandas=2.1
```
in conda way, this is expressing what you want here - any `2.1` would fit, but nothing above (this is what single `=` means in _conda_, unlike pip where it doesn't have a single equals)

---

Wouldn't it be better to use the same approach for conda and pip to make it easier to change versions in the future?

---

It might be better to use unified approach, I agree.